packages/stark-ui/src/modules/breadcrumb/components/breadcrumb.component.scss

```diff
@@ -0,0 +1,52 @@
+/* ============================================================================== */
+/*  S t a r k   B r e a d c r u m b                                               */
+/* ============================================================================== */
+/* stark: src/ui/components/breadcrumb/breadcrumb.component.pcss */
+//.stark-breadcrumb {
+//  display: none;
+//}
+
+.stark-breadcrumb {
+  display: block;
+  a {
+    cursor: pointer;
+  }
+}
+
+@media #{$tablet-only-query} {
```

Comment:
In the old stark, it was `@media #{$tablet-query}`. Could you please adapt this ?